iis-wf/iis-wf-export-actionmanager/src/main/java/eu/dnetlib/iis/wf/export/actionmanager/entity/facade/MDStoreFacade.java

```diff
@@ -0,0 +1,23 @@
+package eu.dnetlib.iis.wf.export.actionmanager.entity.facade;
+
+import eu.dnetlib.data.mdstore.DocumentNotFoundException;
+import eu.dnetlib.data.mdstore.MDStoreServiceException;
+
+/**
+ * MDStore service facade.
+ * 
+ * @author mhorst
+ *
+ */
+public interface MDStoreFacade {
+
+    /**
+     * Delivers single record for given MDStore and record identifiers.
+     * @param mdStoreId MDStore identifier
+     * @param recordId record identifier
+     * @throws MDStoreServiceException when general error occurs 
+     * @throws DocumentNotFoundException when record not found 
+     */
+    public String deliverRecord(String mdStoreId, String recordId) throws MDStoreServiceException;
```

Comment:
I guess we would need to ask Italians why they picked such strange name. We could cut it at this point and rename to `fetchRecord()` in facade.